src/OpenTelemetry/Metrics/MetricReader.cs

```diff
@@ -20,8 +20,10 @@ namespace OpenTelemetry.Metrics
 {
     public abstract class MetricReader : IDisposable
     {
-        private AggregationTemporality preferredAggregationTemporality = AggregationTemporality.Both;
-        private AggregationTemporality supportedAggregationTemporality = AggregationTemporality.Both;
+        private const AggregationTemporality CumulativeAndDelta = AggregationTemporality.Cumulative | AggregationTemporality.Delta;
```

Comment:
Follow up on https://github.com/open-telemetry/opentelemetry-dotnet/pull/2342#discussion_r708826103.

I tried Both/Neither/All/None and **none** of them seems logical / good to me. So I stepped back and removed them, and use `Cumulative | Delta`.

Reply:
If it is going to be a common thing used a lot of places we could add on the enum like...

```csharp
    [Flags]
    public enum AggregationTemporality : byte
    {
        /// <summary>
        /// Cumulative.
        /// </summary>
        Cumulative = 0b1,

        /// <summary>
        /// Delta.
        /// </summary>
        Delta = 0b10,

        CumulativeAndDelta = Cumulative | Delta, 
    }
```

Reply:
Given `CumulativeAndDelta = Cumulative BITWISE OR Delta` is weird, I think we can use the current approach (a private const) for now, introducing `AggregationTemporality.CumulativeAndDelta` in the future is still possible given it is not a breaking change.